core/templates/dev/head/components/outcome_editor_directive.html

```diff
@@ -65,13 +65,10 @@
     <div style="clear: both;"></div>
   </div>
 </div>
-
 <div ng-if="!outcome.feedback.getHtml().isEmpty() && displayFeedback">
-  <audio-translations-editor component-name="<[COMPONENT_NAME_FEEDBACK]>"
-                             subtitled-html="outcome.feedback"
-                             on-start-edit="onAudioTranslationsStartEditAction"
-                             on-change="onAudioTranslationsEdited">
-  </audio-translations-editor>
+  <subtitled-html-editor component-name="<[COMPONENT_NAME_FEEDBACK]>"
+                         subtitled-html="outcome.feedback">
```

Comment:
I think you forgot to pass in the 'display' here?

Reply:
Because I did't figure out how to tackle with OutcomeEditorDirective.js , I think this cause problem in Adding audio translations for responses

Reply:
In OutcomeEditorDirective.js there is no service to pass in the argument in subtitled-display,